doc/api/ssl.rst

```diff
@@ -307,8 +307,15 @@ Context objects have the following methods:
 .. py:method:: Context.set_default_verify_paths()
 
     Specify that the platform provided CA certificates are to be used for
-    verification purposes.  This method may not work properly on OS X.
-
+    verification purposes. This method has some caveats related to the
+    binary wheels that cryptography (pyOpenSSL's primary dependency) ships:
+
+    * macOS will only load certificates using this method if the user has
+      the ``openssl@1.1`` Homebrew formula installed in the default location.
+    * Windows will not work.
+    * manylinux1 cryptography wheels will work in pyOpenSSL 17.1.0 and above.
```

Comment:
Maybe we should be clearer "will work on most common linux distributions" or something